packages/search-ui/src/__tests__/queryString.test.js

```diff
@@ -83,12 +41,6 @@ describe("#parse", () => {
   it("will parse an object correctly", () => {
     expect(subject(stringified)).toEqual(parsed);
   });
-
-  it("will parse an app search filters object correctly", () => {
```

Comment:
Didn't want "App Search" specific tests here. Everything is still covered by other tests.